src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import mate.academy.spring.service.OrderService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import mate.academy.spring.service.dto.mapping.impl.response.OrderResponseMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+    private final ShoppingCartService cartService;
+    private final UserService userService;
+    private final OrderResponseMapper orderResponseMapper;
+
+    public OrderController(OrderService orderService,
+                           ShoppingCartService cartService,
+                           UserService userService,
+                           OrderResponseMapper orderResponseMapper) {
+        this.orderService = orderService;
+        this.cartService = cartService;
+        this.userService = userService;
+        this.orderResponseMapper = orderResponseMapper;
+    }
+
+    @PostMapping("/order-complete")
```

Comment:
```suggestion
    @PostMapping("/complete")
```

otherwise, the link will repeat the word "order" 2 times:   /orders/order-complete